spec/integration/core/vapp_spec.rb

```diff
@@ -174,7 +174,7 @@
           "invalid-vapp-template-id",
           @test_params.vdc_1_name
         )
-      }.to raise_error(Fog::Compute::VcloudDirector::Forbidden, "This operation is denied.")
+      }.to raise_error(Fog::Compute::VcloudDirector::Forbidden, /\[ #{uuid_matcher} \] This operation is denied/)
```

Comment:
Does matching the UUID add anything to these tests? To keep the tests simple, we could search for regex matches against just the text, e.g. `/This operation is denied/`. This would also be more resistant to upstream changes.